cmd/immuadmin/immuadmin.go

```diff
@@ -45,8 +45,9 @@ Environment variables:
   IMMUADMIN_ADDRESS=127.0.0.1
   IMMUADMIN_PORT=3322
   IMMUADMIN_MTLS=true`,
+		SilenceUsage:      true,
+		SilenceErrors:     true,
```

Comment:
@mmeloni Same remark as dor `SilenceUsage`: we should probably set it dynamically if it’s possible.